File: Library.java
import java.util.ArrayList;

public class Library {
    private ArrayList<Books> booksList = new ArrayList<>();
    private ArrayList<Users> usersList = new ArrayList<>();
    private ArrayList<SaleBooks> booksSell = new ArrayList<>();
    //Getters y Setters de la clase.
    public ArrayList<Books> getBooksList() {
        return booksList;
    }
    public ArrayList<Users> getUsersList() {
        return usersList;
    }
    public ArrayList<SaleBooks> getBooksSell() {
        return booksSell;
    }

    //Métodos para añadir elementos a las listas correspondientes.
    public void addBook(Books book){
        booksList.add(book);
    }
    public void addUser(Users user){
        usersList.add(user);
    }
    public void addSellBook(SaleBooks sellbook){
        booksSell.add(sellbook);
    }
}

File: Library.py
class Library:
    booksList = []
    usersList = []
    booksSell = []

    @classmethod
    def get_books(cls):
        return cls.booksList

    @classmethod
    def get_users(cls):
        return cls.usersList

    @classmethod
    def get_sell_books(cls):
        return cls.booksSell

    @classmethod
    def add_book(cls, book):
        cls.booksList.append(book)

    @classmethod
    def add_user(cls, user):
        cls.usersList.append(user)

    @classmethod
    def add_sell_book(cls, sell_book):
        cls.booksSell.append(sell_book)